boa_engine/src/builtins/promise/mod.rs

```diff
@@ -0,0 +1,731 @@
+//! This module implements the global `Promise` object.
+
+#![allow(dead_code, unused_results, unused_variables)]
```

Comment:
I think this shouldn't stay here, right? We don't want to silence warnings :)

Reply:
You're absolutely right, forgot to remove this :)